libs/wingsdk/src/target-tf-gcp/app.ts

```diff
@@ -80,7 +81,9 @@ export class App extends CdktfApp {
     if (props.rootConstruct) {
       const Root = props.rootConstruct;
       if (this.isTestEnvironment) {
-        throw new Error("wing test not supported for tf-gcp target yet");
+        throw new NotImplementedError(
```

Comment:
How come only tf-gcp has to throw this error? How is it different from the other targets?

Reply:
it's the only platform without test support